Localization/l10n.lua

```diff
@@ -30,6 +30,22 @@ local supportedLocals = {
     ['koKR'] = true,
 }
 
+function l10n:InitializeLocaleOverride()
+    local overridingLocale = QUESTIE_LOCALES_OVERRIDE.locale
```

Comment:
`QUESTIE_LOCALES_OVERRIDE` needs to be added to `.luacheckrc` otherwise other pipeline will break, as this is an "unknown global"

Reply:
Thanks, added